src/views/Navbar.jsx

```diff
@@ -199,7 +201,13 @@ const Navbar = (props) => {
                 </Link>
                 <div className="dropdown-divider" />
                 <Link className="dropdown-item" to="/login">
-                  Logout
+                  <button
+                  type="button"
+                  className="logout"
+                  onClick={logout}
+                  >
+                    LOG OUT
+                  </button>
```

Comment:
You have a button inside an anchor tag. When the Link tag "transpiles" it is rendered as an anchor tag in the browser. [It is not valid HTML5 syntax](https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5?answertab=active#tab-top) to have a button tag wrapped in an anchor tag.

Reply:
Thank you. I have resolved this.